Dockerfile.builder

```diff
@@ -0,0 +1,37 @@
+FROM centos:7
```

Comment:
Instead of having two Dockerfiles, we should start using multi-staged Dockerfiles.